crates/proto/proto/zed.proto

```diff
@@ -4,2497 +4,2496 @@ package zed.messages;
 // Looking for a number? Search "// current max"
 
 message PeerId {
-    uint32 owner_id = 1;
-    uint32 id = 2;
+  uint32 owner_id = 1;
```

Comment:
Is there a way we can keep the protobuf file formatted with 4 spaces? Feels like that matches up with the rest of our code, which all uses 4 spaces.